diff --git a/build/gulpfile.extensions.ts b/build/gulpfile.extensions.ts
index dafca7f2adde8d268aa9784dedaa54f2bde8c49e..f3c8f061bd7fe0070c298ad6e4be58e4918b8c87 100644
--- a/build/gulpfile.extensions.ts
+++ b/build/gulpfile.extensions.ts
@@ -86,6 +86,9 @@ const compilations = [
 	'extensions/tunnel-forwarding/tsconfig.json',
 	'extensions/typescript-language-features/web/tsconfig.json',
 	'extensions/typescript-language-features/tsconfig.json',
+	'extensions/vibe-agents/tsconfig.json',
+	'extensions/vibe-chandra/tsconfig.json',
+	'extensions/vibe-remote-ssh/tsconfig.json',
 	'extensions/vscode-api-tests/tsconfig.json',
 	'extensions/vscode-colorize-tests/tsconfig.json',
 	'extensions/vscode-colorize-perf-tests/tsconfig.json',
